java/EpochWriter.java

```diff
@@ -243,11 +257,12 @@ public boolean newValues(
 	 *  [the above does not apply if getSanDiegoFeatures is enabled]
 	 */
 	private void writeEpochSummary(
-			LocalDateTime epochStartTime,
+			// ZonedDateTime epochStartTime,
+			Instant epochStartTime,
```

Comment:
Instead of taking a `LocalDateTime` object, take an `Instant` object (represents a specific point in time).